src/App.js

```diff
@@ -1,13 +1,105 @@
 import React, { Component } from 'react'
+import PropTypes from 'prop-types'
```

Comment:
Runtime prop type checking is pretty bad now. It's unnecessary overhead that you're executing on the client side when you can and should do this statically at build time.

[Here's](https://flow.org/en/docs/react/components/) how you would do it with Flow if you wanted to have types, but even just not doing prop types is better: